packages/device-utils/src/modelUtils.ts

```diff
@@ -7,6 +7,27 @@ export enum DeviceModel {
     UNKNOWN = '',
 }
 
+export enum DeviceInternalModel {
```

Comment:
nit: What about `DeviceInternalModel` -> `DeviceModelInternal`

Those keys (T1,TT,T2B1) are just temporary, right? Or this whole enum is temporary and will be exported from Connect later?